menas/ui/components/schema/schemaDetail.controller.js

```diff
@@ -173,7 +173,16 @@ sap.ui.define([
       if (status === 500) {
         const sSchemaType = this.byId("schemaFormatSelect").getSelectedItem().getText();
 
-        MessageBox.error(`Failed to upload new schema. Ensure that the file is a valid ${sSchemaType} schema and try again.`)
+        let errorMessageDetails = "";
+        try {
+          const oRawResponse = JSON.parse(oParams.getParameter("responseRaw"));
+          if (oRawResponse.message) {
+            errorMessageDetails = `\n\nDetails:\n${oRawResponse.message}`;
+          }
+        } catch (unused) {}
```

Comment:
Maybe logging the error would be helpful here: `console.error()`

Reply:
👍  Thanks for going through tese steps! Good to know what is the common pattern for such caes.

Reply:
Added logging. Simulated an HTML response - worked well.